styles/variables.scss

```diff
@@ -1,4 +1,58 @@
 // Color Variables
-$primary-color: #ff9900;
-$secondary-color: #20008d;
-$ternary-color: #dddddd;
+
+//Primary Colors
+$primary-color-50: #ff9900;
```

Comment:
congrats for the work man, I liked a lot :star_struck:

Reply:
Thank you! Let's go!! 🚀